### PR TITLE
SNOW-829187 Add multistatement query example

### PR DESCRIPTION
**Pull request by sfc-gh-pfus**
### Description
Please explain the changes you made here.

### Checklist
- [ ] Code compiles correctly
- [ ] Run ``make fmt`` to fix inconsistent formats
- [ ] Run ``make lint`` to get lint errors and fix all of them
- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: cmd/multistatement/Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+include ../../gosnowflake.mak
+CMD_TARGET=multistatement
```
[the remark this answers is not in the document]
Comment (sfc-gh-pfus) [reply cmd/multistatement/Makefile:2]: Probably, but this is not the case of this PR. I added another example following the patterns of the existing ones.

File: cmd/multistatement/multistatement.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		if value := os.Getenv(key); value != "" {
+			return value
+		}
+		if failOnMissing {
+			log.Fatalf("#{key} environment not set")
+		}
+		return ""
+	}
+
+	account := env("SNOWFLAKE_TEST_ACCOUNT", true)
```
[the remark this answers is not in the document]
Comment (sfc-gh-pfus) [reply cmd/multistatement/multistatement.go:26]: Nope, this function is defined above.

File: cmd/multistatement/multistatement.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	user := env("SNOWFLAKE_TEST_USER", true)
+	password := env("SNOWFLAKE_TEST_PASSWORD", true)
+	host := env("SNOWFLAKE_TEST_HOST", false)
+	portStr := env("SNOWFLAKE_TEST_PORT", false)
+	protocol := env("SNOWFLAKE_TEST_PROTOCOL", false)
+
+	port := 443
```
[the remark this answers is not in the document]
Comment (sfc-gh-pfus) [reply cmd/multistatement/multistatement.go:33]: Yes, we do, but I see reasoning in this. Now, one interested in particular feature has everything in one file.

File: cmd/multistatement/multistatement.go [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	sf "github.com/snowflakedb/gosnowflake"
+)
+
+func getDSN() (string, *sf.Config, error) {
```
[the remark this answers is not in the document]
Comment (sfc-gh-pfus) [reply cmd/multistatement/multistatement.go:15]: Data source name.

**Review by sfc-gh-dheyman-1 — APPROVED** [review]

File: cmd/multistatement/multistatement.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (sfc-gh-pfus) [reply cmd/multistatement/multistatement.go:33]: I agree ;) But nevertheless - not in this commit.

File: cmd/multistatement/multistatement.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (sfc-gh-pfus) [reply cmd/multistatement/multistatement.go:15]: No, this name is very widely used:
```
❯ grep -ir dsn | wc -l
     469
```

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Comment by sfc-gh-pfus** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**